java/src/jmri/jmrit/display/layoutEditor/LayoutEditorAction.java

```diff
@@ -27,6 +27,8 @@ public void actionPerformed(ActionEvent e) {
         for (int i = 2; i < 100; i++) {
             if (InstanceManager.getDefault(EditorManager.class).contains(name)) {
                 name = "My Layout " + i;
+            } else {
```

Comment:
Could you say a few words about why this is here?  It seems to defeat the purpose of the loop, but perhaps I don't understand what this loop is meant to do.

Reply:
It's looking for a unique layout name… If it finds an existing one with the same name then it bumps the name and tries again… Original code tried all numbers 2 thru 100 (no break). The reality is that it only needs the first one that didn't already exist.

Reply:
Thanks!  Got it.  Was confused by contains() instead of equals().